components/datetime/src/time_zone.rs

```diff
@@ -34,9 +34,17 @@ pub(crate) enum ResolvedNeoTimeZoneSkeleton {
     SpecificLong,
     GmtShort,
     GmtLong,
-    IsoBasic,
-    IsoExtended,
     Bcp47Id,
+    Isox,
```

Comment:
nit: docs maybe? people don't know the x/X notation

Reply:
I'll add docs in the next PR